site/sass/components/_video-card.scss

```diff
@@ -0,0 +1,51 @@
+.video-card {
+  text-decoration: none;
+
+  &__thumbnail-wrapper {
+    border-radius: 0.625rem;
+    display: grid;
+    overflow: hidden;
+    position: relative;
+
+    // Use the same aspect ratio as YouTube does for thumbnails to minimize letterboxing.
+    &::before {
+      content: '';
+      display: block;
+      padding-top: 56.25%;
+      width: 100%;
+    }
```

Comment:
Looks like this comment got missed in the merge conflicts.

Can we use `aspect-ratio` here if it's supported? You can use the same TODO that I've got [here](https://github.com/chromeos/chromeos.dev/blob/e2808f28c73473d689623bc93f70bc2d97b6d8d6/site/sass/components/_embed-responsive.scss), or use that class directly if you'd like, for iOS < 15